tests/integration/_cases/monitors/monitors-run-tokena-auth-win.trycmd

```diff
@@ -0,0 +1,8 @@
+```
+$ unset SENTRY_AUTH_TOKEN
+$ SENTRY_DSN=abcd sentry-cli monitors run foo-monitor -- cmd.exe /C echo 123
+? success
+  WARN    [..] In the future only DSN Auth will be supported for monitor checkins.
```

Comment:
As mentioned in #1537, when using monitors slugs you CANNOT use Token based auth, as this does not give us context to which organization your monitor is part of.

The plan is to disallow token based authentication completely for monitor checks in the future.